dockstore-common/src/license/THIRD-PARTY.properties

```diff
@@ -0,0 +1,64 @@
+# Generated by org.codehaus.mojo.license.AddThirdPartyMojo
+#-------------------------------------------------------------------------------
+# Already used licenses in project :
+# - Apache 2
+# - Apache 2.0
+# - Apache 2.0 License
+# - Apache License 2.0
+# - Apache License, Version 2.0
+# - Apache License, version 2.0
+# - Apache Software License - Version 2.0
+# - Apache Software Licenses
+# - Apache-2.0
+# - BSD 3-clause
+# - BSD New
+# - BSD New license
+# - BSD-2-Clause
+# - CC0
+# - CDDL + GPLv2 with classpath exception
+# - CDDL 1.1
+# - CDDL+GPL License
+# - CDDL/GPLv2+CE
+# - Common Public License Version 1.0
+# - EDL 1.0
+# - Eclipse Distribution License - v 1.0
+# - Eclipse Public License - Version 1.0
+# - Eclipse Public License - v 1.0
+# - Eclipse Public License 1.0
+# - GNU Lesser General Public License
+# - GPL2 w/ CPE
+# - LGPL 2.1
+# - MIT
+# - MIT License
+# - MIT license
+# - MPL 1.1
+# - New BSD License
+# - The Apache License, Version 2.0
+# - The Apache Software License, Version 2.0
+# - The MIT License
+# - Three-clause BSD-style
+#-------------------------------------------------------------------------------
+# Please fill the missing licenses for dependencies :
+#
+#
+#Fri Apr 03 14:05:39 PDT 2020
+org.broadinstitute--wdl-draft2_2.12--41=WDL License https://github.com/openwdl/wdl/blob/master/LICENSE
```

Comment:
Any way to wild card the version? 
Otherwise, we might need something to fail the build if an "Unknown license" creeps in again if someone upgrades the version and thus gets past this file

Reply:
Also is there any automated way to create these IDs? Or was this done by hand? 

This and the above questions, ideally answered via code comment

Reply:
I was unable to find a way to use wildcards for version numbers
IDs can be created using the command:
mvn license:add-third-party -Dlicense.useMissingFile; which creates a THIRD-PARTY.properties file. Then the license information has to be filled in by hand. Then the command mvn license:aggregate-add-third-party is used to aggregate the license information into THIRD-PARTY-LICENSES.txt

Reply:
Added the tag <failOnMissing>true</failOnMissing> which is a flag to fail the build if at least one dependency was detected without a license.

Reply:
if there is a missing license file the command mvn clean install will provide a warning listing the artifact with the missing license. the artifact and the appropriate license string can be added to the THIRD-PARTY-LICENSE.properties file

Reply:
It may be too severe, so others may disagree, but is it possible to make it an error instead of warning? I believe the build is full of warnings, and if the build doesn't actually fail, they may get overlooked.

Reply:
I misspoke, the build does fail and a message tells what the artifact with the missing license file is.